1/alpine/Dockerfile

```diff
@@ -32,40 +35,44 @@ RUN set -x \
 		perl \
 		perl-io-socket-ssl \
 		perl-utils \
+	; \
 	\
-	&& wget -O memcached.tar.gz "$MEMCACHED_DOWNLOAD_URL" \
-	&& echo "$MEMCACHED_SHA1  memcached.tar.gz" | sha1sum -c - \
-	&& mkdir -p /usr/src/memcached \
-	&& tar -xzf memcached.tar.gz -C /usr/src/memcached --strip-components=1 \
-	&& rm memcached.tar.gz \
+	wget -O memcached.tar.gz "$MEMCACHED_URL"; \
+	echo "$MEMCACHED_SHA1  memcached.tar.gz" | sha1sum -c -; \
+	mkdir -p /usr/src/memcached; \
+	tar -xzf memcached.tar.gz -C /usr/src/memcached --strip-components=1; \
+	rm memcached.tar.gz; \
 	\
-	&& cd /usr/src/memcached \
+	cd /usr/src/memcached; \
 	\
-	&& ./configure \
+	gnuArch="$(dpkg-architecture --query DEB_BUILD_GNU_TYPE)"; \
+	./configure \
 		--build="$gnuArch" \
```

Comment:
This is a bug we didn't catch before now because we weren't using `set -u` :joy: :sweat_smile: